app/views/items/create.js.erb

```diff
@@ -0,0 +1,6 @@
+<% if @item.valid? %>
+   $('.js-items').append("<%= escape_javascript(render(@item)) %>");
```

Comment:
I had a look at the previous checkpoints (Creating/ Destroying with Javascript) to implement this and realised I am not sure where this syntax comes from? Is this just the standard way of using the JS? Is there a resource I can read to see other JS syntax?